src/instance/instanceinterfaces.ts

```diff
@@ -24,6 +27,13 @@ export interface IRiskAssessmentValue {
   comment: string | undefined;
 }
 
+const IRiskAssessmentValueObject: IRiskAssessmentValue = {
+  assessments: (Joi.object().pattern(Joi.string(), Joi.number()).required() as unknown) as {},
```

Comment:
I would prefer the full type here for `assessments` instead of just "any object" (`{}`) as `{}` would match basically everything and not warn if the original type changed:
```suggestion
  assessments: (Joi.object().pattern(Joi.string(), Joi.number()).required() as unknown) as { [dimensionId: string]: number },
```